### PR TITLE
Add logger task.

### PR DESCRIPTION
**Pull request by KonovalovaVV**

### CODE REVIEW

File: Logger/Logger/ConsoleLogger.cs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿namespace Logger
+{
+    public class ConsoleLogger: ILogger
```
Comment (MaksHladki) [inline Logger/Logger/ConsoleLogger.cs]: Where is the implementation of this interface?

File: Logger/Logger/DbLogger.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    public class DbLogger : ILogger
+    {
+        public readonly string ConnectionString;
+
+        public DbLogger(string connectionString)
+        {
+            ConnectionString = connectionString;
+        }
+
+        public void Error(string message)
```
Comment (MaksHladki) [inline Logger/Logger/DbLogger.cs]: What is the difference between `Error`, `Warning` and `Info` methods? They are all similar, please reuse the code.

File: Logger/Logger/FileLogger.cs [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    public class FileLogger : ILogger
+    {
+        public readonly string FileName;
+
+        public FileLogger(string fileName)
+        {
+            FileName = fileName;
+        }
+
+        public void Error(string message)
```
Comment (MaksHladki) [inline Logger/Logger/FileLogger.cs]: Please, reuse the code between `Error`, `Warning`, and `Info` methods.

File: Logger/Logger/ILogger.cs [hunk 4]
```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+namespace Logger
+{
+    public interface ILogger
```
Comment (MaksHladki) [inline Logger/Logger/ILogger.cs]: Why do you use a console logger as a default implementation of the interface?

File: Logger/Logger/Program.cs [hunk 5]
```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace Logger
+{
+    public class Program
+    {
+        static void Main(string[] args)
+        {
```
Comment (MaksHladki) [inline Logger/Logger/Program.cs]: Please, add tests other logger types such as DbLogger, FileLogger....

File: Logger/Logger/ConsoleLogger.cs (hunk 1 above)
Comment (KonovalovaVV) [reply Logger/Logger/ConsoleLogger.cs]: Add.

File: Logger/Logger/DbLogger.cs (hunk 2 above)
Comment (KonovalovaVV) [reply Logger/Logger/DbLogger.cs]: Add ExecuteCommand(string message) method.

File: Logger/Logger/FileLogger.cs (hunk 3 above)
Comment (KonovalovaVV) [reply Logger/Logger/FileLogger.cs]: Add ShowMessage(string message) method.

File: Logger/Logger/ILogger.cs (hunk 4 above)
Comment (KonovalovaVV) [reply Logger/Logger/ILogger.cs]: "If there are no logging destinations provided, logger should write logs to console".

File: Logger/Logger/Program.cs (hunk 5 above)
Comment (KonovalovaVV) [reply Logger/Logger/Program.cs]: Add.

**Pull request merged**